### PR TITLE
 Translation of community-contributions.md to hindi

### PR DESCRIPTION
**Pull request by monizb**
Hey,
I have translated the `docs/contributing/community-contributions.md` as i thought it would be useful for new contributors, please provide feedback, thanks!

### CODE REVIEW

File: docs/contributing/community-contributions.md [hunk 1]
```diff
@@ -2,22 +2,23 @@
 title: Community Contributions
```
Comment (arshadkazmi42) [inline docs/contributing/community-contributions.md]: `Community Contributions` => `कम्युनिटी कंट्रिब्यूशंस`

File: docs/contributing/community-contributions.md [hunk 2]
```diff
@@ -2,22 +2,23 @@
...
 ---
 
-A big part of what makes Gatsby great is each and every one of you in the community. Your contributions enrich the Gatsby experience and make it better every day.
+इस समुदाय के मे प्रत्येक जन Gatsby को महान बनाने में एक बड़ा भाग रखता है. आपका योगदान Gatsby अनुभव को समृद्ध करता है और इसे हर दिन बेहतर बनाता है|
```
Comment (arshadkazmi42) [inline docs/contributing/community-contributions.md]: `समुदाय के मे` => `समुदाय के`

Comment (arshadkazmi42) [inline docs/contributing/community-contributions.md]: `रखता है.` => `रखते हैं।`

Comment (arshadkazmi42) [inline docs/contributing/community-contributions.md]: `|` => `।`

change this everwhere, in complete document

File: docs/contributing/community-contributions.md [hunk 3]
```diff
@@ -2,22 +2,23 @@
...
 
-We welcome all contributions from you in the community, and would be thrilled to amplify your voice. Contributions are not limited to code, and can take all shapes and forms:
```
Comment (arshadkazmi42) [inline docs/contributing/community-contributions.md:8]: Remove line break at line 7

File: docs/contributing/community-contributions.md [hunk 4]
```diff
@@ -2,22 +2,23 @@
...
 
-- Your wonderful Gatsby.js website!
-- Gatsby starters
-- Gatsby plugins
-- Blog posts and case studies
-- Talk and lecture videos
-- Workshop materials
-- GitHub issues and pull requests
+हम समुदाय में आप सभी के योगदान का स्वागत करते हैं, और हम आपके आवाज़ को बढ़ने में रोमांच रखते है| योगदान देना सिर्फ कोड तक ही सीमित नहीं है, और यह सब तरह के आकार और रूप ले सकता है जैसे:
```
Comment (arshadkazmi42) [inline docs/contributing/community-contributions.md]: `आपके आवाज़` => `आपकी आवाज़`

Comment (arshadkazmi42) [inline docs/contributing/community-contributions.md]: `रोमांच`

does not seems correct here. Any other thoughts?

Comment (arshadkazmi42) [inline docs/contributing/community-contributions.md]: `| ` => `।`

File: docs/contributing/community-contributions.md [hunk 5]
```diff
@@ -2,22 +2,23 @@
...
 
-...and anything else you can think of. Our showcase and library contribution docs are a great place to start:
+- आपका अद्भुत Gatsby.js वेबसाइट!
```
Comment (arshadkazmi42) [inline docs/contributing/community-contributions.md]: `आपका` => `आपकी`

File: docs/contributing/community-contributions.md [hunk 6]
```diff
@@ -2,22 +2,23 @@
...
+- Gatsby स्टार्टर्स
+- Gatsby प्लगिन्स
+- ब्लॉग पोस्ट्स और केस स्टडीज़
+- बात और व्याख्यान वीडियोस
```
Comment (arshadkazmi42) [inline docs/contributing/community-contributions.md]: `बात और व्याख्यान` => `टॉक और लेक्टर्स`

File: docs/contributing/community-contributions.md [hunk 7]
```diff
@@ -2,22 +2,23 @@
...
+- कार्यशाला सामग्री
+- GitHub इश्यूज और पुल्ल रिक्वेस्ट्स
 
-- [Submit to Site Showcase](/contributing/site-showcase-submissions)
-- [Submit to Starter Library](/contributing/submit-to-starter-library/)
-- [Submit to Plugin Library](/contributing/submit-to-plugin-library/)
+...और कुछ भी जिसके बारे आप सोच सकते हैं. हमारे शोकेस और लाइब्रेरी कॉन्ट्रिब्यूशन डॉक्स शुरुआत करने के लिए एक शानदार जगह है:
```
Comment (arshadkazmi42) [inline docs/contributing/community-contributions.md]: `हैं.` => `हैं।`

File: docs/contributing/community-contributions.md [hunk 8]
```diff
@@ -2,22 +2,23 @@
...
 
-Peruse the rest of our contributor docs and reach out to us on [Twitter](https://twitter.com/gatsbyjs), [Discord](https://gatsby.dev/discord), or in a [GitHub issue](/contributing/how-to-file-an-issue/) if you have any questions about contributing!
+- [साइट शोकेस को सबमिट करें](/contributing/site-showcase-submissions)
```
Comment (arshadkazmi42) [inline docs/contributing/community-contributions.md]: `को सबमिट` => `में सबमिट`

File: docs/contributing/community-contributions.md [hunk 9]
```diff
@@ -2,22 +2,23 @@
...
+- [स्टार्टर लाइब्रेरी को सबमिट करें](/contributing/submit-to-starter-library/)
```
Comment (arshadkazmi42) [inline docs/contributing/community-contributions.md]: `को सबमिट` => `में सबमिट`

File: docs/contributing/community-contributions.md [hunk 10]
```diff
@@ -2,22 +2,23 @@
...
+- [प्लगइन लाइब्रेरी को सबमिट करें](/contributing/submit-to-plugin-library/)
```
Comment (arshadkazmi42) [inline docs/contributing/community-contributions.md]: `को सबमिट` => `में सबमिट`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Great work.
Added some feedback.
We will get this merged once those changes are done

File: docs/contributing/community-contributions.md (hunk 4 above)
Comment (monizb) [reply docs/contributing/community-contributions.md]: @arshadkazmi42 
How about `रूचि`?

**Comment by monizb** [comment]
@arshadkazmi42 
Hey,
Thanks for the review, i have made the requested changes and also added `रूचि`, please let me know if thats fine, thanks :)

File: docs/contributing/community-contributions.md (hunk 4 above)
Comment (arshadkazmi42) [reply docs/contributing/community-contributions.md]: this sounds better

**Comment by arshadkazmi42** [comment]
Great work. 
Please do no mark the feedback as resolved. The reviewer will mark it as resolved after re-checking the changes done by you. You can read more about review process here.

https://github.com/gatsbyjs/gatsby-hi/blob/master/contibuting.md

**Review by arshadkazmi42 — APPROVED** [review]
👍

**Comment by monizb** [comment]
> Great work.
> Please do no mark the feedback as resolved. The reviewer will mark it as resolved after re-checking the changes done by you. You can read more about review process here.
> 
> https://github.com/gatsbyjs/gatsby-hi/blob/master/contibuting.md

Thank you for the clarification, will take care next time :)

**Pull request merged**